05-UI-ReactJS/react-app/src/components/EditFormMovie.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const EditFormMovie = (props) => (
+    <div className="input-group">    
+        <input className="form-control" type="text" value={props.title} onChange={props.onEditMovies} name="title" pattern="^[A-Za-z0-9\s]+$" title="Letters only, please" minLength="2" maxLength="30" required/>
```

Comment:
If you have to many attributes you could write one per line to improve readability